src/Controller/PressPacksController.php

```diff
@@ -101,8 +103,11 @@ public function pressPackAction(Request $request, string $id) : Response
 
         $arguments['relatedContent'] = $arguments['package']
             ->then(function (PressPackage $package) {
-                return ListingTeasers::basic($package->getRelatedContent()->map($this->willConvertTo(Teaser::class, ['variant' => 'secondary']))->toArray());
-            });
+                return $package->getRelatedContent();
+            })
+            ->then(Callback::emptyOr(function (Sequence $articles) {
```

Comment:
This could be simplified by wrapping the original in `Callback::methodEmptyOr()`.